blocks/header/header.css

```diff
@@ -267,9 +405,84 @@ header nav .nav-sections ul > li > ul > li {
   header nav .nav-sections > ul > li > ul > li {
     padding: 8px 0;
   }
+
+  header nav .nav-sections ul > li > a {
+    border-left: 0;
+    padding: 30px 0;
+    font-size: 1rem;
+  }
+
+  header nav .nav-sections ul > li > a::after{
+    content: "";
+    position: absolute;
+    bottom: -32px;
+    left: 0;
+    width: 0;
+    height: 3px;
+    text-decoration: none;
+    border-bottom: 4px solid var(--button-color);
+  }
+  
+  header nav .nav-sections ul li.active > a, header nav .nav-sections ul > li > a:hover {
+    border-left: 0;
+  }
+
+  header nav .nav-sections ul li.active > a::after {
+    width: 100%;
+  }
+  
+  header nav .nav-sections ul > li > a:hover::after{
+    width: 100%;
+  }
 }
 
 /* tools */
 header nav .nav-tools {
   grid-area: tools;
+  background-color: var(--button-color);
+  height: 100%;
```

Comment:
nitpick: there's a `font-weight: 700;` on the "Log in" button on the orginial page that makes it look bolder than what we have

Reply:
done